### PR TITLE
docs: Add instructions to run conformance tests

### LINKED ISSUE #89: Define and document process for running conformance tests
The same as we have for lokomotive-kubernetes, we should have a documented way of running conformance tests, perhaps on all platforms.

### PR DESCRIPTION
**Pull request by surajssd**
Fixes #89

### CODE REVIEW

File: docs/run-conformance-tests.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Follow the canonical document [here](https://github.com/cncf/k8s-conformance/blob/master/instructions.md) which instructs on installing sonobuoy and running tests.
+
+## AWS
+
+For AWS you need to make sure that the node ports are allowed in the security group. To do so run following commands which adds a snippet to the file [`security.tf`](https://github.com/kinvolk/lokomotive/blob/master/assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/security.tf).
+
+```tf
+echo '
+resource "aws_security_group_rule" "worker-nodeport" {
```
Comment (invidian) [inline docs/run-conformance-tests.md:15]: I think we should make this as a configuration flag, so one does not have to modify the source code.

File: docs/run-conformance-tests.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+# Conformance tests
+
+This document enumerates the way steps to run conformance tests for various platforms supported by Lokomotive.
+
+## Running conformance tests
```
Comment (invidian) [inline docs/run-conformance-tests.md:5]: This section should be last, after you have the cluster running.

File: docs/run-conformance-tests.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Conformance tests
+
+This document enumerates the way steps to run conformance tests for various platforms supported by Lokomotive.
```
Comment (invidian) [inline docs/run-conformance-tests.md:3]: We should have a reference to quick start document or configuration example, to have simple instruction how to create conforming cluster.

File: docs/run-conformance-tests.md (hunk 1 above)
Comment (surajssd) [reply docs/run-conformance-tests.md]: done.

File: docs/run-conformance-tests.md (hunk 3 above)
Comment (surajssd) [reply docs/run-conformance-tests.md]: done.

File: docs/run-conformance-tests.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+# Conformance tests
+
+This document enumerates the steps required to run conformance tests for various platforms supported by Lokomotive.
+
+**Note**: There is only one caveat to consider when running tests for AWS. For other platforms you can run conformance tests without making special arrangements.
+
+## AWS
+
+For AWS you need to make sure that the node ports are allowed in the security group. To do so make sure you set the variable `expose_nodeports` to `true` in the AWS config. Read more about this flag in the [AWS reference docs](configuration-reference/platforms/aws.md).
```
Comment (invidian) [inline docs/run-conformance-tests.md:9]:
```suggestion
For AWS you need to make sure that the node ports are allowed in the security group. To do so make sure you set the `expose_nodeports` cluster property to `true` in the AWS config. Read more about this flag in the [AWS reference docs](configuration-reference/platforms/aws.md).
```

File: docs/run-conformance-tests.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+
+This document enumerates the steps required to run conformance tests for various platforms supported by Lokomotive.
+
+**Note**: There is only one caveat to consider when running tests for AWS. For other platforms you can run conformance tests without making special arrangements.
+
+## AWS
+
+For AWS you need to make sure that node ports are allowed in the security group. To do so make sure you set the `expose_nodeports` cluster property to `true` in the AWS config. Read more about this flag in the [AWS reference docs](configuration-reference/platforms/aws.md).
+
+To install the cluster on AWS follow [AWS quick start guide](quickstarts/aws.md).
```
Comment (iaguis) [inline docs/run-conformance-tests.md:11]:
```suggestion
To install the cluster on AWS follow the [AWS quick start guide](quickstarts/aws.md).
```

Comment (surajssd) [reply docs/run-conformance-tests.md]: No problem :-)

**Pull request merged**